server/main.lua

```diff
@@ -1467,3 +1506,12 @@ function GetVehicleInformation(plate)
     end
 end
 
+function GetVehicleOwner(plate, owner)

```

Comment:
```suggestion
function GetVehicleOwner(plate)

```
`owner` arg is not used here either